### DISCUSSION TITLE
Is it compatible with Samsung watch7?

### DISCUSSION THREAD

**Discussion by wuhongsheng** [ITDev93/SHM-MOD#5] [General]

**Comment by ITDev93** [comment]
Yes

**Comment by wuhongsheng** [comment]
Why should applications be whitelisted?

**Comment by ITDev93** [comment]
> Why should applications be whitelisted?

Google put restrictions on what apps can run in the background for the new WearOS... 

System apps by default can run without issues, but normal apps don't have that...

By default on WearOS watches, battery optimization is aggressive, it kills services, thus, no IHRN or Sleep Apnea, these run in background...

The WearOS lacks of interface for battery optimization, unlike our phones, we can choose which applications to turn on or off the battery optimization, so the only way to do it is via ADB command, and as you can see, i updated the companion to do that for new install, and i added buttons to run the commands, just incase the command was not successful after install (very unlikely to happen)...
